src/pages/Strategies/Strategies.tsx

```diff
@@ -18,14 +26,35 @@ const Strategies = (): JSX.Element => {
           transaction fees!
         </P>
       </Card>
-      <StyledList>
-        {Object.values(StrategyType).map((strategyType) => (
-          <StrategyCard key={strategyType} strategyType={strategyType} />
-        ))}
-        <Card alignItems='center' justifyContent='center'>
-          <P size='xs'>More Strategies coming soon</P>
-        </Card>
-      </StyledList>
+      {strategies ? (
+        <StyledList>
+          {strategies.map((strategy) => {
+            const { title, summary } = getContent(strategy, t);
+
+            const to = PAGES.STRATEGY.replace(`:${URL_PARAMETERS.STRATEGY.TYPE}`, strategy.type);
+
+            return (
+              <Link key={strategy.type} to={to}>
```

Comment:
Moved the Link out of of the Card component it is simply a visual element with no logic in it or interaction.